src/core/test/fuzzing/src/test/scala/FuzzingTest.scala

```diff
@@ -24,74 +24,34 @@ class FuzzingTest extends TestBase {
 
   test("Verify stage fitting and transforming") {
     val exemptions: Set[String] = Set(
-      "com.microsoft.ml.spark.TrainClassifier",
-      "com.microsoft.ml.spark.TrainRegressor",
-      "com.microsoft.ml.spark.PartitionSample",
       "org.apache.spark.ml.feature.FastVectorAssembler",
-      "com.microsoft.ml.spark.Repartition",
-      "com.microsoft.ml.spark.UnrollImage",
-      "com.microsoft.ml.spark.SummarizeData",
-      "com.microsoft.ml.spark.CheckpointData",
-      "com.microsoft.ml.spark.ComputePerInstanceStatistics",
-      "com.microsoft.ml.spark.ComputeModelStatistics",
-      "com.microsoft.ml.spark.DataConversion",
-      "com.microsoft.ml.spark.ImageTransformer",
-      "com.microsoft.ml.spark.IndexToValue",
-      "com.microsoft.ml.spark.CNTKLearner",
-      "com.microsoft.ml.spark.FindBestModel",
-      "com.microsoft.ml.spark.ImageFeaturizer",
-      "com.microsoft.ml.spark.CNTKModel",
       "com.microsoft.ml.spark.ValueIndexerModel",
-      "com.microsoft.ml.spark.PartitionSample",
-      "com.microsoft.ml.spark.Cacher",
-      "com.microsoft.ml.spark.ImageSetAugmenter",
-      "com.microsoft.ml.spark.EnsembleByKey",
-      "com.microsoft.ml.spark.ClassBalancer",
-      "com.microsoft.ml.spark.Timer")
+      "com.microsoft.ml.spark.CNTKLearner",
+      "com.microsoft.ml.spark.TrainClassifier",
```

Comment:
what were the issues for the ones that are left?

Reply:
My laziness lol